### PR TITLE
Feature/string attribute details

### PR DESCRIPTION
**Pull request by kgoedecke**
*Description of changes:*
- Adds kerning, strikethrough and underline for string attribute.
- Adds text transform (uppercase, lowercase)
- Add support for fixed and auto sized text fields.
- Support for line height

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kgoedecke** [comment]
@dbanksdesign I added a few more things. Hope it's ok to do it in the same PR.

File: models/StringAttribute/StringAttribute.js [hunk 1]
```diff
@@ -56,11 +56,17 @@ class StringAttribute {
...
           MSAttributedStringColorAttribute: new Color(args.color),
           textStyleVerticalAlignmentKey: verticalAlignmentMap[args.verticalAlignment || 'top'],
+          underlineStyle: args.underline ? 1 : 0,
+          strikethroughStyle: args.strikethrough ? 1 : 0,
+          kerning: args.kerning || undefined,
           paragraphStyle: {
             _class: 'paragraphStyle',
             alignment: textAlignmentMap[args.alignment || 'left'],
+            ...(args.lineHeight && { maximumLineHeight: args.lineHeight }),
+            ...(args.lineHeight && { minimumLineHeight: args.lineHeight }),
```
Comment (L2jLiga) [inline models/StringAttribute/StringAttribute.js]: may be ?
```javascript
...(args.lineHeight
 ? {
   maximumLineHeight: args.lineHeight,
   minimumLineHeight: args.lineHeight,
 }
 : {}),
```

Comment (dbanksdesign) [reply models/StringAttribute/StringAttribute.js]: The way @kgoedecke has it is a bit more readable to me, but I'm ok either way.

Comment (L2jLiga) [reply models/StringAttribute/StringAttribute.js]: For me to, let @kgoedecke choose

Comment (kgoedecke) [reply models/StringAttribute/StringAttribute.js]: Yeah, not the biggest fan of multiline `: ?`. Let's stick with this for now.

Comment (L2jLiga) [reply models/StringAttribute/StringAttribute.js]: Ok

**Comment by kgoedecke** [comment]
@dbanksdesign feel free to more this guy in :-)

**Pull request merged**